README.md

```diff
@@ -1,203 +1,86 @@
-# vet 
-## Automate Open Source Package Vetting in CI/CD
-
-`vet` is a tool for identifying risks in open source software supply chain. It
-helps engineering and security teams to identify potential issues in their open
-source dependencies and evaluate them against organizational policies.
-
+<h1 align="center">
+    vet
+</h1>
+<p align="center">
+    🙌 Refer to <b><a href="https://safedep.io/docs/">https://safedep.io/docs</a></b> for the documentation 📖
+</p>
+
+![License](https://img.shields.io/github/license/safedep/vet)
+![Release](https://img.shields.io/github/v/release/safedep/vet)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/safedep/vet/badge)](https://api.securityscorecards.dev/projects/github.com/safedep/vet)
 [![CodeQL](https://github.com/safedep/vet/actions/workflows/codeql.yml/badge.svg?branch=main)](https://github.com/safedep/vet/actions/workflows/codeql.yml)
 [![Scorecard supply-chain security](https://github.com/safedep/vet/actions/workflows/scorecard.yml/badge.svg)](https://github.com/safedep/vet/actions/workflows/scorecard.yml)
 
-## Demo
-
-![vet Demo](docs/images/vet-demo.gif)
```

Comment:
Should we keep this demo gif here? I have  seen a lot of repo that keeps a demo in their README so that users get a quick feel of what it does.